TARS/src/mcu_main/telemetry.h

```diff
@@ -68,6 +69,25 @@ struct TelemetryPacket {
     int16_t barometer_temp;   //[-128, 128]
 };
 
+struct CompactTelemetryPacket {
+    unsigned short idx = 1;
+    float gps_lat;
+    float gps_long;
+    float gps_alt;
+    uint8_t voltage_battery;  //[0, 16]
+    int16_t barometer_temp;   //[-128, 128]
+};
+
+struct Packets {
+    enum {
+        COMPACT,
```

Comment:
nit: Since in the future we may define more packet types, let's make the names a bit more descriptive. For example, COMPACT could be named PAD_LANDED and DEFAULT could be named MAIN_FLIGHT etc.